SerializedData.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.IO;
+using UnityEngine;
+
+namespace DUCK.PersistentDataStore
+{
+	/// <summary>
+	/// A persistent data store for serializable objects, identified by their type, or optionally by a unique ID.
+	/// </summary>
+	public static class SerializedData
```

Comment:
I realise that this is names `SerializedData` but is there a more generic version for when I wanna store some bytes into a file? I am guessing the `File` API already supports this. Which begs the question - what does this module achieve beyond wrap the `File` API?

Reply:
There isn't a more generic version at the moment since it was only ever needed for serialized data so far - the main advantage over the File API is that it's restricted to working within Unity's persistent data path, and automatically separates the file locations by UID if supplied. It's saved me a lot of time on Montessori when a task says 'make something persistent / save X to the device', I literally just call this API and I know it's sorted. I can't really think of an application use-case which wouldn't benefit from persistent file management being wrapped in this way.